schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_polymer_entity_instance.json

```diff
@@ -115,6 +115,38 @@
                      }
                   ]
                },
+               "connect_type": {
```

Comment:
I wonder now if we should wait for adding this until search requirements are clearly defined. See comment on connectivity type in https://docs.google.com/document/d/178DBKN31Nq9QS0ENvY1Ixt0GVNTn0dUoRA4L_Ha39lM/edit

Reply:
Ah, perhaps. Are there drawbacks to pushing it out now and adjusting the search context later?

Reply:
Or rather, is it possible to release it in the schema now, but just refrain from loading it? This would certainly be the easier and safer route on my end, as regenerating the schemas will require a lot of changes. It would be very easy for me to just turn off loading, though.

Reply:
I would rather postpone. As we discussed, this is not going to be useful for search as it's a per-bond annotation

Reply:
OK understand, I'll carry things back. Thanks for following up.